### DISCUSSION TITLE
I'm curious about how Phosphor Icons adds to overall bundle size

### DISCUSSION THREAD

**Discussion by flyingace** [phosphor-icons/homepage#581] [General]
Let's say I was using 10 icons, each only in one format, from the `@phosphor-icons/react `package. How much should I expect this to increase my bundle size? How might this compare to my adding 10 svg files for the same icons downloaded directly from phosphoricons.com? How much (in bytes/kilobytes) does the core of this library add to my bundle?

**Comment by rektdeckard** [comment]
The current implementation of `@phosphor-icons/react` is tree-shakeable but includes all 6 weights of each icon imported, regardless of whether the weights are used or not. How this adds up is:

- ~4-5kb per icon imported
- 2kb shared library code

So I would expect 10 icons to add roughly 45-55kb to your bundle size, depending on the complexity of the icons.
